app/celery/process_ses_receipts_tasks.py

```diff
@@ -333,6 +334,7 @@ def process_ses_smtp_results(
                     created_at=headers['date'],
                     status=notification_status,
                     reference=ses_message['mail']['messageId'],
+                    notification_id=uuid4(),
```

Comment:
Why are you creating an id and passing it in here? That seems to be handled in `persist_notification`. Are we never going to have a `request_id` from utils when this code is called?

Reply:
When setting g.request_id in auth it made the utils change irrelevant for service-authenticated requests, but still applies to admin tasks.